build_runner/lib/src/server/hot_reload_client.dart.js

```diff
@@ -0,0 +1,3857 @@
+// Generated by dart2js (full emitter, strong), the Dart to JavaScript compiler version: 2.0.0-dev.69.4.
```

Comment:
we should check if there are other flags we should be using. --minify, maybe --omit-implicit-checks

ideally we should get this code size down